spec/display-buffer-spec.coffee

```diff
@@ -703,6 +715,11 @@ describe "DisplayBuffer", ->
       expect(displayBuffer.screenPositionForBufferPosition([100000, 0])).toEqual [12, 2]
       expect(displayBuffer.screenPositionForBufferPosition([100000, 100000])).toEqual [12, 2]
 
+    it "clips correctly around atomic tab characters", ->
+      buffer.insert([0, 0], '\t')
```

Comment:
Wait, this should be independent of tab length. We're going from buffer position, which is either 0 (before the tab) or 1 (after the tab), and the result should be either 0 in the first case or tabLength in the second case.